content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -133,79 +119,194 @@ Examples of common errors indicating that there is an issue with trusting a TLS/
 
-    ![image-20201018155224381](../../../docs/doc_imgs/reference/certificate-verification-expire-date.png)
+1. Re-check the timezones of the events in the query - sometimes the time appears in the event in the UI is different than the one in the 3rd party due to time zone differences, ensure the times point to the same TZ.
+2. Double check for any filter params in the configurations - sometimes the integration configuration will contain filter params - an open text/dropdown filter to a specific field optional values, etc.. if you see events missing in the UI, double check those events doesn’t match the existing filters in the integration configuration page. In particular, make sure the filter does not point to past time.
+3. Make sure suspicious missing events cannot be found using their unique ID to search the incidents tab.
 
-    * Resolution: If the certificate expired, make sure to renew the certificate at the target endpoint.
+### Fetch mechanism stopped fetching new events
 
-  * Common name / Alt name -  A certificate signed only for a specific URI, For example, if the certificate is signed for `test.com` and the integration is accessing the endpoint using `test1.com` the certification validation will fail.
+#### Check the fetch history
 
-    * Identify: `Endpoint certificate` -> `Subject` -> `CommonName` and `certificate` -> `Extentions` -> `SubjectAlternativeName`:
+Check the [fetch history](#fetch-history) for the following:
 
-      ![image-20201018160939173](../../../docs/doc_imgs/reference/certificate-verification-altnames.png)
+1. Temporary error from the API - Sometimes fetch mechanism can encounter temporary issues when calling the API, such issues will usually appear once or twice before getting back to fetch normally and will contain the 500 (internal server) error.
+2. Params collision - In some integrations, there are params that can’t be configured together and will throw an error if they do. The error should appear in the fetch history and should be informative about which two params can’t be configured together.
+3. Error from the API related to malformed params - Sometimes an error can come up from the API informing us that there was an issue with the params passed as part of the request, this error usually occurs in integrations where there’s a free-text filter param that might be malformed, double check all params are correct and ensure the error message doesn’t point to such issue.
+4. Error from the API, getting 429 (rate limit) error - Some APIs have a rate limiter in their system configured to a certain value that might be reached. when getting 429 try to increase the fetch interval as it will result less calls in a certain amount of time, and raise the limit as it usually affect the number of calls due to the data being fetched in less time.
 
-      ![image-20201018160950403](../../../docs/doc_imgs/reference/certificate-verification-common-name.png)
+#### Reset last run
```

Comment:
this also - could be a section header but it should be mostly used as a bullet point to try

Reply:
There's fetch which is the section (denoted by ##)
Then there's Fetch mechanism stopped fetching new events which is the problem (denoted by ###)
Then there's check fetch history and reset last run which are the proposed troubleshooting steps (denoted by ####)